flask_api/app/routes.py

```diff
@@ -0,0 +1,176 @@
+    DEFAULT_TREES_DISTANCE,
+)
+from . import db
+from .helpers import utils
+from .helpers.generate_trees import TreesGenerator
+from .models import User, Orchard
+
+api = Blueprint("api", __name__)
+
+
+@api.route("/healthchecker", methods=["GET"])
+def healthchecker():
+    return jsonify({"status": "I am alive!"}), 200
+
+
+@api.route("/users", methods=["POST"])
+def create_user():
+    name = request.form.get(USERNAME_KEY)
+    if not name:
+        return jsonify({"status": "Name can't be empty."}), 400
+
+    password = request.form.get(PASSWORD_KEY)
+    if not password:
+        return jsonify({"status": "Password can't be empty."}), 400
+
+    is_name_duplicated = User.query.filter_by(username=name).first()
+    if is_name_duplicated:
+        return (
+            jsonify(
+                {"status": "This name is already used, please use a different one."}
+            ),
+            400,
+        )
+
+    user = User(username=name)
+    user.set_password(password)
+    db.session.add(user)
+    db.session.commit()
+
+    return jsonify({"status": f"{name} saved to the database."})
+
+
+@api.route("/users/database", methods=["POST"])
+def show_database():
+    users = User.query.all()
+    orchards = Orchard.query.all()
+    return jsonify({"status": [{"users": f"{users}"}, {"orchards": f"{orchards}"}]})
+
+
+@api.route("/users/database/clear", methods=["POST"])
+def clear_database():
+    db.drop_all()
+    db.create_all()
+    db.session.commit()
+    return jsonify({"status": "Database clear!"})
+
+
+@api.route("/users/orchards", methods=["POST"])
+def create_orchard():
+    if not current_user.is_authenticated:
+        return jsonify({"status": "You have to login first!"}), 400
+
+    coords = request.files.get(COORDS_KEY)
+    distance_between_rows = request.form.get(ROWS_KEY)
+    distance_between_trees = request.form.get(TREES_KEY)
+
+    if not coords:
+        return jsonify({"status": "Failed! Coordinates can't be empty!"}), 400
+
+    if not coords.filename.endswith(".csv"):
+        return (
+            jsonify({"status": "Failed! Coordinates file must have a .csv extension!"}),
+            400,
+        )
+
+    if not distance_between_rows:
+        distance_between_rows = DEFAULT_ROWS_DISTANCE
+    else:
+        try:
+            distance_between_rows = float(distance_between_rows)
+        except ValueError:
+            return (
+                jsonify(
+                    {"status": "Failed! Distance between rows must be a float number!"}
+                ),
+                400,
+            )
+
+    if not distance_between_trees:
+        distance_between_trees = DEFAULT_TREES_DISTANCE
+    else:
+        try:
+            distance_between_trees = float(distance_between_trees)
+        except ValueError:
+            return (
+                jsonify(
+                    {"status": "Failed! Distance between trees must be a float number!"}
+                ),
+                400,
+            )
+
```

Comment:
Consider using some kind of schemas to specify the requirements of the request structure. For example: [Marshmallow](https://marshmallow.readthedocs.io/en/stable/api_reference.html?highlight=float#marshmallow.fields.Float)